int8 pan_deg
int8 tilt_deg
---
int8 pan_deg
int8 tilt_deg
